src/en/federation/federation.md

```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
 - `https://lemmy.ml/post/123` (Post)
 - `https://lemmy.ml/comment/321` (Comment)
 
-You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
\ No newline at end of file
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean: 
+- new posts
+- new comments
+- votings
+- edited comments and posts
+
+You can copy the URL of the community from the address-bar in your browser and insert it in your search-field. Wait a few seconds, the post will appear below. There is no indicator for a running search now!
```

Comment:
For comments, the format is `https://lemmy.ml/comment/456`, so you cant copy it directly and have to remove  the `/post/123` part first.